src/Marten/Events/AggregateToExtensions.cs

```diff
@@ -15,7 +15,7 @@ public static class AggregateToExtensions
         /// <param name="state"></param>
         /// <typeparam name="T"></typeparam>
         /// <returns></returns>
-        public static T AggregateTo<T>(this IMartenQueryable<IEvent> queryable, T state = null) where T : class
```

Comment:
I would suggest you to add new methods as below rather than changing the signature of the existing ones since it results in breaking changes in terms of binary compatibility
```cs
public static T AggregateTo<T>(this IQueryable<IEvent> queryable, T state = null) where T : class
{
    return AggregateTo(queryable.As<IMartenQueryable<IEvent>>(), state);
}

public static async Task<T> AggregateToAsync<T>(this IQueryable<IEvent> queryable, T state = null,
    CancellationToken token = new()) where T : class
{
    return await AggregateToAsync(queryable.As<IMartenQueryable<IEvent>>(), state, token).ConfigureAwait(false);
}
```

Reply:
Will do.